package.json

```diff
@@ -166,7 +166,7 @@
     "stylelint": "10.1.0",
     "stylelint-config-standard": "18.3.0",
     "vue-cli-plugin-pug": "1.0.7",
-    "vue-template-compiler": "2.6.10",
+    "vue-template-compiler": "2.6.3",
```

Comment:
Why was this downgraded? Please restore the latest.

Reply:
according to Pierre on Slack, this is because they upgraded the compiler but failed to upgrade vue for some reason, and the versions are supposed to match.

Reply:
It need to match vue version